curp/src/server/cmd_worker/conflict_checked_mpmc.rs

```diff
@@ -190,8 +191,8 @@ enum OnceState {
 /// Internally it maintains a dependency graph of conflicting cmds
 
 struct Filter<C: Command, CE> {
-    /// Index from `ProposeId` to `vertex`
-    cmd_vid: HashMap<ProposeId, u64>,
+    /// Index from `LogIndex` to `vertex`
+    cmd_vid: HashMap<LogIndex, u64>,
```

Comment:
Will there be truncation and log overwriting behavior in Raft, and will this have an impact?

Reply:
For leader node, logs will not be overwritten.
For follower node, only uncommitted logs can be overwritten. and  uncommitted logs will not send to mpmc conflict channel